sdk/applicationinsights/azure-resourcemanager-applicationinsights/src/main/java/com/azure/resourcemanager/applicationinsights/fluent/models/ApplicationInsightsComponentProperties.java

```diff
@@ -72,7 +72,7 @@ public final class ApplicationInsightsComponentProperties
     private OffsetDateTime creationDate;
 
     /*
-     * Azure Tenant Id.
+     * Azure Tenant Id. The values allowed for `tenant-id` are: `common`, `organizations`, `consumers`, or the tenant ID.
```

Comment:
Files in this module is generated from Swagger.

Please also update there https://github.com/Azure/azure-rest-api-specs/blob/main/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-02-02/components_API.json#L535-L539

Otherwise, this change will be overridden in next update.

Reply:
Thanks @weidongxu-microsoft ， after confirming that only the files under the sdk/spring directory need to be modified, so revert the changes to this file.